doc/howto/bundle.rst

```diff
@@ -0,0 +1,40 @@
+How to Bundle Resources
+=======================
+
+This guide will show you how to configure Dune to generate modules with string resources
+from other files in your project.
+
+Folder Structure
+----------------
+
+.. code:: bash
+
+    $ tree src
+    src
+    └── lib
+        └── my_lib
+            ├── dune
+            └── resources
+                └── site.css
```

Comment:
IMO this can be simplified to
```
    lib
    ├── dune
    └── resources
        └── site.css
```
no need to invoke `tree`, or show two unnecessary levels.

Reply:
That's there for people, who like me, need an additional pointer on where the `resources` go in relation to the project root and the `dune` file. If you disagree with it though, feel free to create a PR removing it!